src/context_store/storage/redis.py

```
@@ -0,0 +1,94 @@
+                    await self._redis.delete(*keys_to_delete)
+                    keys_to_delete.clear()
+            if keys_to_delete:
+                await self._redis.delete(*keys_to_delete)
+        except Exception as exc:
+            logger.warning("Redis invalidate_prefix failed (degraded): %s", exc)
+
+    async def clear(self) -> None:
+        """Remove all cache entries."""
+        try:
+            await self._redis.flushdb()
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)